django/http/response.py

```diff
@@ -383,7 +384,10 @@ def streaming_content(self):
     @streaming_content.setter
     def streaming_content(self, value):
         # Ensure we can never iterate on "value" more than once.
-        self._iterator = iter(value)
+        if hasattr(value, 'read'):
```

Comment:
The other option is to wrap the file with something like:

    from django.core.files import File
    class ChunkedFile(File):
        def __iter__(self):
            return self.chunks()
    return StreamingResponse(ChunkedFile(open(fullpath, 'rb')))